share/lib/common.sh

```diff
@@ -27,10 +27,10 @@ service_action() {
   service="$1"
   action="$2"
 
-  if [ -x /usr/sbin/service ]; then
-    CMD="service ${service} ${action}"
-  elif [ -x /etc/init.d/${service} ]; then
+  if [ -x /etc/init.d/${service} ]; then
```

Comment:
yes but when the service is founs, there is a formated line that starts with the service name that appears